plugins/woocommerce/client/admin/client/settings-payments/onboarding/components/stepper/index.tsx

```diff
@@ -118,7 +125,7 @@ export default function Stepper( {
 								key={ step.id }
 								label={ step.label }
 								isCompleted={ isStepCompleted( step ) }
-								isActive={ step.id === active }
+								isActive={ step.id === activeTopLevelStep }
```

Comment:
The sidebar only cares about top-level active step, not sub-steps

Reply:
I so love these verbose variable names.